lib/dul_arclight/traject/ead2_config.rb

```diff
@@ -120,15 +121,27 @@
 #end
 
 to_field 'title_filing_si', extract_xpath('/ead/eadheader/filedesc/titlestmt/titleproper[@type="filing"]')
-to_field 'title_ssm', extract_xpath('/ead/archdesc/did/unittitle')
-to_field 'title_formatted_ssm', extract_xpath('/ead/archdesc/did/unittitle', to_text: false)
+to_field 'title_ssm' do |record, accumulator|
+  result = record.xpath('/ead/archdesc/did/unittitle')
+  result = result.collect do |n|
+    n.xpath('.//text()[not(ancestor::unitdate)]').collect(&:text).join(" ")
+  end.join(" ")
+  accumulator << result
+end
+to_field 'title_formatted_ssm' do |record, accumulator|
+  whole_title = record.xpath('/ead/archdesc/did/unittitle').to_a
+  no_dates = whole_title.collect do |parent_node|
+    parent_node.children.select { |elem| elem.name != 'unitdate' }
+  end.flatten
+  accumulator.concat no_dates
```

Comment:
ugh I started typing a response and then github _hid the conversations on the changed files page_ and I can't figure out how to get them back or if my draft message is still there. Anyway. The goal is precisely to put the actual nodes into the accumulator, in case there are formatting tags in the title. (The original code is using `to_text: false` for this purpose.)

Some repositories are putting `<unitdate>` tags inside their `<unittitles>` but this must have not been the case for any of the test EADs we were using in the summer 2019 community development cycle. There is a point later down the line in which the "normalized title" (I think) is created by concatenating the title with the date, so we were ending up with things like "Issues of _Example_ magazine, 1920s-1970s, bulk 1931-1965, 1920s-1970s, bulk 1931-1965"